### ISSUE TITLE
Support for ambertools23

### ISSUE THREAD

**Issue by xiki-tempula** [openmm/openmmforcefields#327]
ambertools23 has been there for a year now and we are interested in using some new options from the ambertools23 which are not available in ambertools22. I wonder if it is possible for openmmforcefields to release a version that supports ambertools23? Thanks.

**Comment by mattwthompson** [comment]
I don't think this is currently being worked on https://github.com/openmm/openmmforcefields/issues/280

**Comment by mattwthompson** [comment]
Release 0.13.0 (upcoming today with some luck, next week otherwise) drops GAFF support but allows AmberTools 23 to be installed alongside it

**Comment by xiki-tempula** [comment]
When ambertools24 is released, will it be a easy update?
